tests/resources/helper_functions.py

```diff
@@ -285,7 +285,7 @@ def send_mock_usdc_to_address(
     mock_usdc = ERC20Token(web3, get_address_of_type(config, "MockUSDC"))
     initial_recipient_balance = mock_usdc.balanceOf(recipient)
 
-    if mock_usdc.balanceOf(factory_deployer) >= amount:
+    if mock_usdc.balanceOf(factory_deployer.address) >= amount:
```

Comment:
Was this a bug?

Reply:
Yes, it required the address, not the wallet object.